test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/Helpers/ProcessHelperTests.cs

```diff
@@ -1,171 +1,168 @@
-        }
-
-        private class TestableProxyOperationManager : ProxyOperationManager
-        {
-            public TestableProxyOperationManager(
-                ITestRequestSender requestSender,
-                ITestRuntimeProvider testHostManager,
-                int clientConnectionTimeout,
-                int errorLength) : base(requestSender, testHostManager, clientConnectionTimeout)
-            {
-                base.ErrorLength = errorLength;
-            }
-        }
-
-        private class TestableTestHostManager : DefaultTestHostManager
-        {
-            public TestableTestHostManager(
-                Architecture architecture,
-                Framework framework,
-                IProcessHelper processHelper,
-                bool shared,
-                int errorLength) : base(architecture, framework, processHelper, shared)
-            {
-                base.TimeOut = 30000;
-                base.ErrorLength = errorLength;
-                base.Initialize(TestSessionMessageLogger.Instance);
-            }
-
-            public override TestProcessStartInfo GetTestHostProcessStartInfo(IEnumerable<string> sources, IDictionary<string, string> environmentVariables, TestRunnerConnectionInfo connectionInfo)
-            {
-                return new TestProcessStartInfo();
-            }
-        }
-        private class TestableProcessHelper : IProcessHelper
-        {
-            private string ErrorMessage;
-
-            public void SetErrorMessage(string errorMessage)
-            {
-                this.ErrorMessage = errorMessage;
-            }
-            public string GetCurrentProcessFileName()
-            {
-                throw new NotImplementedException();
-            }
-
-            public int GetCurrentProcessId()
-            {
-                throw new NotImplementedException();
-            }
-
-            public string GetTestEngineDirectory()
-            {
-                throw new NotImplementedException();
-            }
-
-            public Process LaunchProcess(string processPath, string arguments, string workingDirectory, IDictionary<string, string> envVariables, Action<Process, string> errorCallback)
-            {
-                var process = Process.GetCurrentProcess();
-
-                errorCallback(process, this.ErrorMessage);
-                errorCallback(process, this.ErrorMessage);
-                errorCallback(process, this.ErrorMessage);
-                errorCallback(process, this.ErrorMessage);
-
-                return process;
-            }
-
-            public string GetProcessName(int processId)
-            {
-                throw new NotImplementedException();
-            }
-        }
-    }
-}
+//// Copyright (c) Microsoft Corporation. All rights reserved.
+//// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+//namespace TestPlatform.CrossPlatEngine.UnitTests
```

Comment:
Yes, all the test in ProcessHelperTest were earlier written to verify ErrorCallBack method, since then it has been moved to individual Runtime providers